Bottom: 8ddf10564a2d8fea149a4f29b204f8274decb177
Top:    873f4ba92f6e10f4a1b4cb9a9dc9db63de29c025
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-12 09:25:56 -0500

Refresh of buildbot-worker-host

---
diff --git a/ansible-playbook/staging b/ansible-playbook/staging
index e544fd42..6c319766 100644
--- a/ansible-playbook/staging
+++ b/ansible-playbook/staging
@@ -3,6 +3,10 @@ testdaedalus.novatech-llc.com
 
 [docker-hosts]
 testdaedalus.novatech-llc.com
+buildbot-worker.novatech-llc.com
 
 [buildsystem]
 buildsystem.novatech-llc.com
+
+[buildbot-worker]
+buildbot-worker.novatech-llc.com
